inputfiles/addedTypes.json

```diff
@@ -2713,5 +2713,11 @@
                 "type": "boolean"
             }
         ]
+    },
+    {
+        "kind": "property",
+        "interface": "Document",
+        "name": "onvisibilitychange",
+        "type": "(this: Document, ev: Event) => any"
```

Comment:
really `any` or better `void`?

Reply:
@HolgerJeromin I just followed the pattern I saw other event listeners on the Document type, it appeared the other type definitions had `any` as the return type.

If you think `void` is more appropriate happy to make that change.

Reply:
Oh, i have not checked that. I would keep it consistent, too :)